pgcontents/tests/test_pgmanager.py

```diff
@@ -207,11 +213,16 @@ def test_rename_directory(self):
     def test_max_file_size(self):
 
         cm = self.contents_manager
-        max_size = 68
+        max_size = 120
```

Comment:
These changed because encryption adds a header overhead to the data written to the database.